guide/how-it-works/payment-request-formats.md

```diff
@@ -19,119 +19,235 @@ images:
 
-This identifier builds on top of LNURL-Pay and introduces the familiar format of email addresses as identifiers.
+##### Lightning address
 
-## Examples
+A [Lightning addresses](https://lightningaddress.com) builds on top of LNURL-Pay and gives users a more human readable payment address. They use the very familiar email format (user@bitcoin.design) to request payments.
 
-Each request format has its own unique approach to bundling payment information for transmission. Unique are Lightning addresses, which are easily readable and memorizable by users.
+##### Withdraw
 
-{% include fact/open.html color="blue" label="On-chain address" icon="triangle" %}
+LNURL-withdraw allows a user to pull a payment from another user. These are useful when offering refunds or if the request is needed to be generated by the sender for some reason.
 
-<span style="word-break: break-word;">bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq</span>
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/payment-request-formats/lnurl.jpg"
+   retina = "/assets/images/guide/how-it-works/payment-request-formats/lnurl@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/payment-request-formats/lnurl-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/payment-request-formats/lnurl-mobile@2x.jpg"
+   alt-text = "How LSPs offer liquidity as a service"
+   width = 800
+   height = 348
+%}
+
+### Node ID
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/payment-request-formats/node.jpg"
+   retina = "/assets/images/guide/how-it-works/payment-request-formats/node@2x.jpg"
+   alt-text = "A QR code of an offer"
+   width = 400
+   height = 500
+   layout = "float-right-desktop"
+%}
 
-{% include fact/close.html %}
+A users node ID can be used as a payment request format when a sender is paying via [Keysend](https://lightning.readthedocs.io/lightning-keysend.7.html) or an [Atomic multi-path (AMP)]({{ '/guide/how-it-works/payment-request-formats/#atomic-multi-path-amp' | relative_url }}) payment request.
 
-{% include fact/open.html color="blue" label="Lightning invoice (BOLT 11)" icon="triangle" %}
+Node IDs can act as static payment end points. This means the user does not have to generate a request each time they want to receive a payment.
 
-<span style="word-break: break-word;">LNBC1PVJLUEZSP5ZYG3ZYG3ZYG3ZYG3ZYG3ZYG3ZYG3ZYG3ZYG3ZYG3ZYG3ZYG3ZYGSPP5QQQSYQCYQ5RQWZQFQQQSYQCYQ5RQWZQFQQQSYQCYQ5RQWZQFQYPQDPL2PKX2CTNV5SXXMMWWD5KGETJYPEH2URSDAE8G6TWVUS8G6RFWVS8QUN0DFJKXAQ9QRSGQ357WNC5R2UEH7CK6Q93DJ32DLQNLS087FXDWK8QAKDYAFKQ3YAP9US6V52VJJSRVYWA6RT52CM9R9ZQT8R2T7MLCWSPYETP5H2TZTUGP9LFYQL</span>
+Requesting a payment with your node ID can leak your public information which is bad for privacy. Keysend payments are also unable to provide proof of payment like an invoice payment which could be problematic in some situations.
```

Comment:
```suggestion
Requesting a payment with your node ID can leak your public information, which is bad for privacy. Also, unlike invoice payments, keysend payments cannot provide proof of payment. This could be problematic in some situations.
```
Added a needed comma. Rearranged for simplicity.